internal/sampling/http/thrift-0.9.2/constants.go

```diff
@@ -15,9 +15,11 @@ import (
 )
 
 // (needed to ensure safety because of naive import list construction.)
-var _ = thrift.ZERO
-var _ = fmt.Printf
-var _ = bytes.Equal
+var (
```

Comment:
can we undo these? They should be excluded already from linting / fmt, and I don't want extra changes recorded in these files